modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -6,28 +6,34 @@ endif::[]
 
 :imagesdir: ../assets/images
 
```

Comment:
Please consider or apply the edits to the previous .adoc file to this one.
Much of the test is the same.
I have copyedited only the changed text.